backendRoot/src/test/java/com/virtuslab/gitmachete/backend/integration/GitMacheteStatusTest.java

```diff
@@ -36,47 +42,74 @@
 
   GitMacheteRepositoryBuilder gitMacheteRepositoryBuilder = new GitMacheteRepositoryBuilder(repositoryDir);
 
-  @Before
```

Comment:
The annotations are no longer needed... b/c we rely now instead on `init`/`cleanup` JUnit's default names for tests, right?

Reply:
No, AFAIK JUnit doesn't have any default names for tests. I removed this annotation coz I want to pass proper parameter to this method (to copy selected script) and I call this methods manually in test cases.